### PR TITLE
Add config options, headers

### PR DESCRIPTION
**Pull request by dchw**

### CODE REVIEW

File: cloud/client.go [hunk 1]
```diff
@@ -1382,3 +1383,7 @@ func (c *client) UploadLog(pathOnDisk string) (string, error) {
 
 	return fmt.Sprintf(uploadBundleResponse.ViewURL), nil
 }
+
+func (c *client) GetAuthToken() string {
+	return c.authToken
```
Comment (alexcb) [inline cloud/client.go:1388]: should this force a refresh if it's expired?

**Review by alexcb — COMMENTED** [review]
looks good, just one minor question.

File: cloud/client.go (hunk 1 above)
Comment (dchw) [reply cloud/client.go:1388]: We only ever call it right after newing up the client, which IIRC refreshes the token if needed.

File: cloud/client.go [hunk 2]
```diff
@@ -1385,5 +1385,6 @@ func (c *client) UploadLog(pathOnDisk string) (string, error) {
 }
 
 func (c *client) GetAuthToken() string {
+	c.Authenticate() // Ensure the current token is valid
```
Comment (dchw) [inline cloud/client.go:1388]: This will get a new token and write it to disk if needed.

Comment (alexcb) [reply cloud/client.go:1388]: Does this need error handling?

**Review by alexcb — APPROVED** [review]

**Pull request merged**